lib/miq_automation_engine/engine/miq_ae_service_model_legacy.rb

```diff
@@ -0,0 +1,94 @@
+      'orchestration_stack_openstack'                => 'ManageIQ_Providers_Openstack_CloudManager_OrchestrationStack',
+      'miq_provision_openstack'                      => 'ManageIQ_Providers_Openstack_CloudManager_Provision',
+      'security_group_openstack'                     => 'ManageIQ_Providers_Openstack_CloudManager_SecurityGroup',
+      'template_openstack'                           => 'ManageIQ_Providers_Openstack_CloudManager_Template',
+      'vm_openstack'                                 => 'ManageIQ_Providers_Openstack_CloudManager_Vm',
+      'ems_openstack'                                => 'ManageIQ_Providers_Openstack_CloudManager',
+      # Openstack Infra
+      'ems_cluster_openstack_infra'                  => 'ManageIQ_Providers_Openstack_InfraManager_EmsCluster',
+      'host_openstack_infra'                         => 'ManageIQ_Providers_Openstack_InfraManager_Host',
+      'orchestration_stack_openstack_infra'          => 'ManageIQ_Providers_Openstack_InfraManager_OrchestrationStack',
+      'ems_openstack_infra'                          => 'ManageIQ_Providers_Openstack_InfraManager',
+      # Red Hat
+      'host_redhat'                                  => 'ManageIQ_Providers_Redhat_InfraManager_Host',
+      'miq_provision_redhat'                         => 'ManageIQ_Providers_Redhat_InfraManager_Provision',
+      'miq_provision_redhat_via_iso'                 => 'ManageIQ_Providers_Redhat_InfraManager_ProvisionViaIso',
+      'miq_provision_redhat_via_pxe'                 => 'ManageIQ_Providers_Redhat_InfraManager_ProvisionViaPxe',
+      'template_redhat'                              => 'ManageIQ_Providers_Redhat_InfraManager_Template',
+      'vm_redhat'                                    => 'ManageIQ_Providers_Redhat_InfraManager_Vm',
+      'ems_redhat'                                   => 'ManageIQ_Providers_Redhat_InfraManager',
+      # VMware
+      'host_vmware'                                  => 'ManageIQ_Providers_Vmware_InfraManager_Host',
+      'host_vmware_esx'                              => 'ManageIQ_Providers_Vmware_InfraManager_HostEsx',
+      'miq_provision_vmware'                         => 'ManageIQ_Providers_Vmware_InfraManager_Provision',
+      'miq_provision_vmware_via_pxe'                 => 'ManageIQ_Providers_Vmware_InfraManager_ProvisionViaPxe',
+      'template_vmware'                              => 'ManageIQ_Providers_Vmware_InfraManager_Template',
+      'vm_vmware'                                    => 'ManageIQ_Providers_Vmware_InfraManager_Vm',
+      'ems_vmware'                                   => 'ManageIQ_Providers_Vmware_InfraManager',
+      # Others
+      'configuration_manager'                        => 'ManageIQ_Providers_ConfigurationManager',
+      'provisioning_manager'                         => 'ManageIQ_Providers_ProvisioningManager',
+      'ems'                                          => 'ExtManagementSystem'
```

Comment:
It's a convenient alias.  It probably should not be deprecated, but that's up to you @gmcculloug